osu.Game/Rulesets/UI/FrameStabilityContainer.cs

```diff
@@ -207,11 +211,15 @@ private void updateClock()
 
         private void setClock()
         {
-            // in case a parent gameplay clock isn't available, just use the parent clock.
-            parentGameplayClock ??= Clock;
-
-            Clock = GameplayClock;
-            ProcessCustomClock = false;
```

Comment:
Sure you want to omit this?

Are these changes still required at all?

Reply:
yes, see the ProcessFrame method.